app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,2 @@
+class PostsController < ApplicationController
+end
```

Comment:
- [ ] The `Posts` controller should contain two actions, the `index` action and the `show` action. It is alright if they are empty for now because Rails will conventionally display whatever is in the `view` files if the action in a controller is empty. 👍 😎